Tests/RxCocoaTests/UITabBarItem+RxTests.swift

```diff
@@ -15,17 +15,36 @@ final class UITabBarItemTests : RxTest {
 }
 
 extension UITabBarItemTests {
-    func testBadgetValue_Text() {
```

Comment:
If you have a second to make a PR just for this fix that would be awesome - typos in code are a bummer 😂

Reply:
For sure 😄